packages/Search/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -144,8 +144,7 @@ KOKKOS_FUNCTION int nearestQuery( BVH<DeviceType> const bvh,
     {
         Node const *leaf = TreeTraversal<DeviceType>::getRoot( bvh );
         int const leaf_index = TreeTraversal<DeviceType>::getIndex( bvh, leaf );
-        double const leaf_distance =
-            distance( query_point, leaf->bounding_box );
+        double const leaf_distance = distance( leaf );
```

Comment:
I prefer the old notation. It is strange to compute the distance with only one input. If it wasn't for the diff, I would think that you are computing the distance between the leaf and the origin.

Reply:
`distance` here is a functor that measures the distance to nodes in the tree.\

Reply:
I know. It's just less easy to read than before.  
```C++
distance(query_point, leaf->bounding_box);
```
is clear that what you are computing while
```C++
distance(leaf);
```
is not

Reply:
So is the geometry for which we are querying distance now embedded in the distance functor?

Reply:
@sslattery Exactly.

Reply:
so perhaps this can be resolved with a different naming convention like `distanceToGeometry` to infer that the geometry is in the functor